src/DBFTables.jl

```diff
@@ -452,8 +452,9 @@ function Base.getproperty(dbf::Table, name::Symbol)
     nrow = header.records
     @inbounds field = getfields(dbf)[col]
     str = getstrings(dbf)
-    @inbounds colarr = [julia_value(field, str[col, i]) for i = 1:nrow]
-    return colarr
+    ft = field.type
+    fv = Val{field.dbf_type}()
+    return @inbounds Union{field.type, Missing}[julia_value(ft, fv, str[col, i]) for i = 1:nrow]
```

Comment:
Can we try moving all of this to a function barrier function, where we dispatch on both `ft` and `fv` like:
```julia
function _getcol(::Type{FT}, ::Type{FV}, str, col) where {FT,FV}
    Union{FT, Missing}[julia_value(FT, Val{FV}(), str[col, i]) for i = 1:nrow]
end
```

Reply:
Plus, the version of the `julia_value` function I'm dispatching to now is type stable.

Reply:
Sure its not just that I missing the `@inbounds` or something? But If its type stable now thats fine I guess